docker/Dockerfile.k8s

```diff
@@ -14,6 +14,9 @@
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 #
 # This Dockerfile is incomplete and should only be used with other Dockerfiles
+
+COPY requirements2.k8s.txt ./
```

Comment:
Don‘t think we need this file, we can just install these deps in a common image.  Image spacifically for k8s doen't make sense.
one for normal use, one for cuda is enough.

Reply:
Yea, make sense, gonna merge the k8s dependencies into the main image, I avoided that initially to mitigate image bloating for public images